CHANGELOG

```diff
@@ -8,6 +8,7 @@
 * Adyen: Support for additional AVS code mapping [jknipp] #3236
 * Update message for AVS result code 'A' to generically cover postal code mismatches [jknipp] #3237
 * CyberSource: Update CyberSource SOAP documentation link [vince-smith] #3204
+* Braintree: Add `skip_avs` and `skip_cvv` gateway specific fields [leila-alderman] #3241
```

Comment:
Note you'll need to re-add this afterward if any other commits have been made that update the changelog.